server/routes/immediate-needs-report/immediateNeedsReportController.ts

```diff
@@ -85,20 +100,29 @@ export default class ImmediateNeedsReportController {
       const stateKey = {
         prisonerNumber: prisonerData.personalDetails.prisonerNumber,
         userId: req.user.username,
-        assessmentType,
         pathway,
+        assessmentType,
       }
-      const currentPage = await this.assessmentStateService.getCurrentPage(stateKey)
+
+      const existingWorkingAssessmentVersion = await this.assessmentStateService.getWorkingAssessmentVersion(stateKey)
+      const currentApiAssessmentPage = await this.rpService.getAssessmentPage(
+        prisonerData.personalDetails.prisonerNumber as string,
+        pathway as string,
+        currentPageId,
+        assessmentType,
+        existingWorkingAssessmentVersion || 1,
```

Comment:
Probably one for another ticket, but should the default version be configurable?

Reply:
This default is for when there's no version in the cache which now should be never so we can probably just remove this at some point. I'll create a new task to do some more tidying up in this area.